HISTORY.md

```diff
@@ -1,3 +1,11 @@
+1.15.3 / 2016-12-xx
+===================
+
+  * deps: qs@6.3.0
+    - Date stringification format configurable (https://github.com/ljharb/qs/issues/159)
```

Comment:
We don't use the stringification API, so this is an irrelevant change to our users. Please only list the changes that affect our users.